eos/form-factors/parametric-bgl1997-impl.hh

```diff
@@ -101,6 +103,18 @@ namespace eos
                  UsedParameter(p[_par_name("F2_1")], *this),
                  UsedParameter(p[_par_name("F2_2")], *this),
                  UsedParameter(p[_par_name("F2_3")], *this) }},
+        _a_T1{{  UsedParameter(p[_par_name("T1_0")], *this),
+                 UsedParameter(p[_par_name("T1_1")], *this),
+                 UsedParameter(p[_par_name("T1_2")], *this),
+                 UsedParameter(p[_par_name("T1_3")], *this) }},
+        _a_T2{{  UsedParameter(p[_par_name("T2_0")], *this),
+                 UsedParameter(p[_par_name("T2_1")], *this),
+                 UsedParameter(p[_par_name("T2_2")], *this),
+                 UsedParameter(p[_par_name("T2_3")], *this) }},
```

Comment:
hmm isn't one of these parameters fixed by the endpoint relations?

Reply:
We have implemented the endpoint relations for both the SM and the BSM form factors.